dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -976,7 +1018,11 @@ public void shouldReturnPageHTMLForPersona() throws Exception{
      * @throws Exception
      */
     @Test
-    public void shouldReturnPageHTMLForLegacyUUID() throws Exception {
+    @UseDataProvider("cases")
+    public void shouldReturnPageHTMLForLegacyUUID(final Container container, final Template templateTestCase) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)